app/controllers/dashboard_controller.rb

```diff
@@ -0,0 +1,6 @@
+class DashboardController < ApplicationController
+
+	def index
```

Comment:
make sure your tabs are set to two spaces - this is important (even if it seems nitpicky!)